plugins/nid-team/rebase/rebase_manager.py

```
@@ -0,0 +1,702 @@
+        pr_num = active_pr["number"]
+        log_info(f"Scanning Draft PR #{pr_num} comments for human feedback...")
+        
+        # 1. Fetch comments on PR using gh API
+        try:
+            # Gets review comments & issue comments
+            comments_json = self.run_cmd([
+                "gh", "api", f"repos/{self.repo_name}/issues/{pr_num}/comments",
+                "--json", "id,body,author,createdAt"
+            ])
+            comments = json.loads(comments_json)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)